src/components/measurementTable/MeasurementTable.js

```diff
@@ -149,7 +149,7 @@ class MeasurementTable extends Component {
     return this.props.timepoints.map((timepoint, index) => {
       return (
         <div key={index} className="measurementTableHeaderItem">
-          <div className="timepointLabel">{timepoint.label}</div>
+          <div className="timepointLabel">{this.props.t(timepoint.key)}</div>
```

Comment:
Why are you changing this from `timepoint.label` to `timepoint.key`?

Key is going to be something like baseline, current, comparison, or nadir. We want to display the label or the timepoint, e.g. Baseline, Follow-up 1, Follow-up 2, ..., not the key.

(though I don't know how this is going to show up in Locize...)